src/SessionManagementAPI.ts

```diff
@@ -6,6 +6,7 @@ export interface SessionManagementAPI {
     getUserId(): SocialId
     getDomain(): string
 
+    setProfileInfo(displayName: String, avatarUrl: String): Promise<void>
```

Comment:
I would set the param to the same object as returned before (and reuse the type here) for maintaining compatibility in the future.